### PR TITLE
added react hook usage to documentation for further clarity

### LINKED ISSUE #144: How to get value of inputted date?
Got a question. How do I get the value of the inputted date on the onChange event? 
I'm using react hook
`const [currentDate, setNewDate] = useState("")`
Here's the code
`<SemanticDatepicker onChange={evt => setNewDate(evt.target.value)}/>`

I tried following the example in the Codepen with `onDateChange` but that doesn't seem to work in the new version.

### PR DESCRIPTION
**Pull request by artawood**
This is to add to the documentation an example of react hook for usage and add clarity to how the date is retrieved with params `(event, date)` and displayed from property `value` via `data.value`

Closes #144.

### CODE REVIEW

**Comment by arthurdenner** [comment]
@artawood, your PR is on point, thank you!

I was thinking, could you fix the typo in `data` and replace the two current examples considering a hook usage, please? I think it makes more sense. For example, the basic example can be:

```
const AppWithBasic = () => {
  const [currentDate, setNewDate] = useState('');
  const onChange = (event, data) => setNewDate(data.value);

  return <SemanticDatepicker onChange={onChange} />;
};
```

**Comment by artawood** [comment]
@arthurdenner, typo is fixed, and I updated the two examples with hook usage. I hope the second example is more helpful as it add clarity the expected value retrieved for the range is an array.

File: README.md [hunk 1]
```diff
@@ -45,17 +45,21 @@ yarn add react-semantic-ui-datepickers
...
 
-const AppWithBasic = ({ onChange }) => (
-  <SemanticDatepicker onChange={onChange} />
-);
-
-const AppWithRangeAndInPortuguese = ({ onChange }) => (
-  <SemanticDatepicker locale="pt-BR" onChange={onChange} type="range" />
-);
+const AppWithBasic = () => {
+  const [currentDate, setNewDate] = useState('');
```
Comment (arthurdenner) [inline README.md]:
```suggestion
  const [currentDate, setNewDate] = useState(null);
```

My bad, I didn't pay attention when writing the example for you, but this initial state should be `null`.

**Comment by arthurdenner** [comment]
Thank you for your contribution, @artawood! 🎉

**Comment by arthurdenner** [comment]
@all-contributors please add @artawood for doc.

**Comment by artawood** [comment]
@arthurdenner Thanks! Happy to contribute! 👍

**Pull request merged**